sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Metrics/GetMetricFeedbacksOptions.cs

```diff
@@ -6,15 +6,14 @@
 
 namespace Azure.AI.MetricsAdvisor.Models
 {
-
     /// <summary> The GetMetricFeedbackOptions. </summary>
-    public partial class GetMetricFeedbackOptions
+    public class GetMetricFeedbacksOptions
     {
         internal MetricFeedbackFilter _metricFeedbackFilter;
 
         /// <summary> Initializes a new instance of GetMetricFeedbackOptions. </summary>
         /// <param name="metricId"> filter feedbacks by metric id. </param>
-        public GetMetricFeedbackOptions(string metricId)
+        public GetMetricFeedbacksOptions(string metricId)
```

Comment:
The metric ID is not used anymore, so we can remove the constructor and the property.